tests/tensor/test_math_opt.py

```diff
@@ -4439,3 +4502,27 @@ def test_is_1pexp(self):
             assert is_1pexp(1 + 2 * exp_op(x), False) is None
         finally:
             config.warn__identify_1pexp_bug = backup
+
+
+def test_log1mexp_stabilization():
+    mode = config.mode
+    if mode == "FAST_COMPILE":
+        mode = "FAST_RUN"
+    mode = get_mode(mode)
```

Comment:
When we need to test an optimization/rewrite's logic, we should manually set the compilation mode to use Python&mdash;so that it will run even when a C compiler isn't available&mdash;and include only the relevant rewrites (e.g. use `mode = Mode("py", opts)`).

A test or two may be necessary to determine whether or not a rewrite works in combination with all the other rewrites, so it's not entirely unreasonable to use something like `FAST_RUN`, but try to do most/all of the rewrite-specific testing without the need for other rewrites.  In other words, *rewrite tests should work in isolation*.  When they don't, they tend to be very difficult to understand or update later on.